ENIGMAsystem/SHELL/Universal_System/Resources/polygon_internal.h

```diff
@@ -0,0 +1,135 @@
+/** Copyright (C) 2021 Nabeel Danish
+*** Nabeel Danish
+**/
+
+#ifndef ENIGMA_POLYGON_INTERNAL_H
+#define ENIGMA_POLYGON_INTERNAL_H
+
+#include <vector>
+#include "AssetArray.h"
+
+namespace enigma 
+{
+	#define NULL_POINT -1;
```

Comment:
seems like good place for constexpr instead.

Reply:
This is removed, since there is no use for this